.gitignore

```diff
@@ -1,2 +1,10 @@
+# IntelliJ files
```

Comment:
Would it be worth using the [Python .gitignore template](https://github.com/github/gitignore/blob/master/Python.gitignore) for our Lambda functions?